neo/public/javascripts/map/controllers.js

```diff
@@ -394,56 +399,94 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
             fillOpacity: 0.2
           };
         } else {
-            return {
-          fillColor: getColor(feature.properties.count),
-          weight: 2,
-          opacity: 1,
-          color: 'white',
-          dashArray: '3',
-          fillOpacity: 0.5
+          return {
+            fillColor: getColor(feature.properties.count),
+            weight: 2,
+            opacity: 1,
+            color: 'white',
+            dashArray: '3',
+            fillOpacity: 0.5
           };
         }
       }
 
+      function setNormalizedPropCountText(d){
+        // beautify 0.0000123 => 1.23e-5, 1.123 => 1.1
+        if(d.properties.count < 1){
+          d.properties.countText = d.properties.count.toExponential(1);
+        }
+        else{
+          d.properties.countText = d.properties.count.toFixed(1);
+        }
+        d.properties.countText += "M";
+      }
+
       //FIXME: the code in county and city (and probably the state) levels are quite similar. Find a way to combine them.
-      if ($scope.status.logicLevel == "state" && $scope.geojsonData.state) {
+      if ($scope.status.logicLevel === "state" && $scope.geojsonData.state) {
```

Comment:
A good example taken from https://stackoverflow.com/questions/523643/difference-between-and-in-javascript:
```
0 == false   // true
0 === false  // false, because they are of a different type
1 == "1"     // true, automatic type conversion for value only
1 === "1"    // false, because they are of a different type
null == undefined // true
null === undefined // false
'0' == false // true
'0' === false // false```